src/libxdata/XWorld.cpp

```diff
@@ -133,6 +134,18 @@ bool XWorld::areConnected(std::shared_ptr<world::NavNode> from, const std::share
     return false;
 }
 
+void XWorld::addRegion(const std::string &code) {
+    findOrCreateRegion(code);
+}
```

Comment:
Added this function so that UserFix has a way to create a dummy region for user fixes. It's the only use of the function, so alternative options might be to have each subclass of world create the USER region implicitly, but that approach would lack transparency.